src/DataCollector/CassandraDataCollector.php

```diff
@@ -12,6 +12,13 @@
 class CassandraDataCollector extends DataCollector
 {
     /**
+     * Human readable values for consistency
+     *
+     * @var array
+     */
+    static protected $consistency = ['any', 'one', 'two', 'three', 'quorum', 'all', 'local quorum', 'each quorum', 'serial', 'local serial', 'local one'];
```

Comment:
It would be better to use constants for the indices of this array.

Ex: 
```php
[
  \Cassandra::CONSISTENCY_ANY => 'any',
  \Cassandra::CONSISTENCY_ONE => 'one',
  ...
]
```